src/Mutator/Extensions/MBString.php

```diff
@@ -0,0 +1,262 @@
+use PhpParser\Node;
+
+/**
+ * @internal
+ */
+final class MBString extends Mutator
+{
+    private const MB_CASES = [
+        'MB_CASE_UPPER' => 0,
+        'MB_CASE_LOWER' => 1,
+        'MB_CASE_TITLE' => 2,
+        'MB_CASE_FOLD' => 3,
+        'MB_CASE_UPPER_SIMPLE' => 4,
+        'MB_CASE_LOWER_SIMPLE' => 5,
+        'MB_CASE_TITLE_SIMPLE' => 6,
+        'MB_CASE_FOLD_SIMPLE' => 7,
+    ];
+
+    private $converters;
+
+    public function __construct(MutatorConfig $config)
+    {
+        parent::__construct($config);
+
+        $settings = $this->getSettings();
+
+        $this->setupConverters($settings);
+    }
+
+    /**
+     * @return Node|Node[]|Generator
+     */
+    public function mutate(Node $node)
+    {
+        yield from $this->converters[$this->getFunctionName($node)]($node);
+    }
+
+    protected function mutatesNode(Node $node): bool
+    {
+        return isset($this->converters[$this->getFunctionName($node)]);
+    }
+
+    private function setupConverters(array $functionsMap): void
+    {
+        $converters = [
+            'mb_chr' => $this->mapNameSkipArg('chr', 1),
+            'mb_ereg_match' => $this->mapEreg($this->mapNameSkipArg('preg_match', 2), '^', '', [$this, 'warpEqOne']),
+            'mb_ereg_replace_callback' => $this->mapEreg($this->mapNameSkipArg('preg_replace_callback', 3)),
+            'mb_ereg_replace' => $this->mapEreg($this->mapNameSkipArg('preg_replace', 3)),
+            'mb_ereg' => $this->mapEreg($this->mapName('preg_match'), '', '', [$this, 'warpTernary']),
+            'mb_eregi_replace' => $this->mapEreg($this->mapNameSkipArg('preg_replace', 3), '', 'i'),
+            'mb_eregi' => $this->mapEreg($this->mapName('preg_match'), '', 'i', [$this, 'warpTernary']),
+            'mb_ord' => $this->mapNameSkipArg('ord', 1),
+            'mb_parse_str' => $this->mapName('parse_str'),
+            'mb_send_mail' => $this->mapName('mail'),
+            'mb_split' => $this->mapName('split'),
+            'mb_strcut' => $this->mapNameSkipArg('substr', 3),
+            'mb_stripos' => $this->mapNameSkipArg('stripos', 3),
+            'mb_stristr' => $this->mapNameSkipArg('stristr', 3),
+            'mb_strlen' => $this->mapNameSkipArg('strlen', 1),
+            'mb_strpos' => $this->mapNameSkipArg('strpos', 3),
+            'mb_strrchr' => $this->mapNameSkipArg('strrchr', 2),
+            'mb_strrichr' => $this->mapNameSkipArg('strrchr', 2),
+            'mb_strripos' => $this->mapNameSkipArg('strripos', 3),
+            'mb_strrpos' => $this->mapNameSkipArg('strrpos', 3),
+            'mb_strstr' => $this->mapNameSkipArg('strstr', 3),
+            'mb_strtolower' => $this->mapNameSkipArg('strtolower', 1),
+            'mb_strtoupper' => $this->mapNameSkipArg('strtoupper', 1),
+            'mb_substr_count' => $this->mapNameSkipArg('substr_count', 2),
+            'mb_substr' => $this->mapNameSkipArg('substr', 3),
+            'mb_convert_case' => $this->mapConvertCase(),
```

Comment:
This mutator will not work for `mb_convert_case($subject, $mode)`. It will only work for `mb_convert_case($subject, MB_CASE_UPPER)`.  I will add support for `mb_convert_case($subject, 1)`

Reply:
support for `mb_convert_case($subject, 1)` added